src/GHC/RTS/Events/Binary.hs

```diff
@@ -103,7 +104,7 @@ getEvent (EventParsers parsers) = do
      then return Nothing
      else do !evTime   <- get
              evSpec <- parsers ! fromIntegral etRef
-             return $ Just Event { evCap = undefined, .. }
+             return $ Just Event { evCap = Nothing, .. }
```

Comment:
I don't know why it was undefined previously but why is this change needed?

Reply:
I was trying to print out this value at a point where it was undefined. I will revert it.